bin/circleci/test-frontend.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 set -ex
 
+npm run content
```

Comment:
since the app now uses `experiments.json`directly we need to build it in order to run the tests